src/TesApi.Web/BatchScheduler.cs

```diff
@@ -447,7 +447,8 @@ private static string StandardizeStartTaskPath(string startTaskPath, string defa
         /// <param name="inputFile"><see cref="Tes.Models.TesInput"/> file</param>
         /// <returns>True if the file is a Cromwell command script</returns>
         private static bool IsCromwellCommandScript(TesInput inputFile)
-            => inputFile.Name.Equals("commandScript");
+            // See https://github.com/broadinstitute/cromwell/blob/17efd599d541a096dc5704991daeaefdd794fefd/supportedBackends/tes/src/main/scala/cromwell/backend/impl/tes/TesTask.scala#L58
+            => inputFile.Name.Equals("commandScript") && inputFile.Description.EndsWith(".commandScript") && inputFile.Type == TesFileType.FILEEnum;
```

Comment:
Why is the name not enough? What else could commandScript be?

Reply:
If the caller is Cromwell, that is sufficient to determine which file is the command script. However, this will false positive for non-cromwell callers that also happen to name their command script "commandScript", and we are using this to try to disambiguate not just the command script, but if it came from cromwell.